### ISSUE TITLE
"there are known memory leaks in this version"

### ISSUE THREAD

**Issue by KarlSevenSeven** [twinbasic/twinbasic#1561]
"there are known memory leaks in this version"

Why is this around since a couple of betas?
I thought if a memory leak is _known_, one can fix it.

So far I haven't had problems with the memory leaks.
But there's always a bad feeling that saving might fail, or destroy the project completely, or whatever.

**Comment by WaynePhillipsEA** [comment]
They are known about, in the sense that you can observe them quite clearly (using taskmgr or the IDE setting to show resource usage).

No reasonable effort has yet been made into further investigating them, as there are higher priorities at the moment.

Note that the leaks are only inside the IDE, and do not affect compiled projects (... unless the leaks are preventing a full compile)

**Comment by wqweto** [comment]
I think this should probably be made to sound even scarier so that people stop expecting that the compiler, the editor or even the runtime to behave as one polished retail product i.e. exceeding the results by lowering the expectations.

There are known leaks, bugs and yesterday a report came someone's drive C: was not available anymore :-))

**Comment by GaryMiller** [comment]
Could this be what is causing my error?  I am unable to get past this error.
15:11:48.268[DEBUGGER] global variables have been cleared
15:11:48.285 Executing '[MyForm].Show()'...
15:11:48.6200001 22:11:48.518 NATIVE EXCEPTION: ACCESS_VIOLATION  {unknown}
15:11:49.618restarting from MEMORY [C:\Users\aiguy\OneDrive\Documents\ZandraTB\ZandraTB.twinproj]

**Comment by WaynePhillipsEA** [comment]
@GaryMiller no, it's unlikely.  That's more likely something that I will need to debug here, if you can share the twinproj.
